libr/io/p/io_r2k.c

```diff
@@ -482,10 +558,10 @@ static int run_ioctl_command (RIO *io, RIODesc *iodesc, const char *buf) {
 
 			for (i = 0; i < map_data.n_entries; i++) {
 				struct kernel_map_info *in = &info[i];
-				io->cb_printf ("start_addr: 0x%"PFMT64x"\n", in->start_addr);
-				io->cb_printf ("end_addr: 0x%"PFMT64x"\n", in->end_addr);
+				io->cb_printf ("start_addr: 0x%x\n", in->start_addr);
```

Comment:
use PFMT64x for addresses. always. and cast to (ut64) if that value can be 32 or 64 to expand it to 64